configure.ac

```diff
@@ -401,6 +401,9 @@ AC_SUBST(LIYESCRYPT)
 AC_CHECK_LIB(crypt, crypt, [LIYESCRYPT=-lcrypt],
 	[AC_MSG_ERROR([crypt() not found])])
 
+AC_CHECK_HEADERS(/usr/include/bsd/readpassphrase.h,
```

Comment:
Hmm, please do not use absolute pathnames, this hardcodes filesystem policy. I'll try to prepare a proposal later today for autodetection using the libbsd-overlay.

Reply:
Hi @guillemj ,

Just a friendly ping, in case you have some time to have a look at this :)

Thanks,
Alex

Reply:
Sorry, right this slipped my mind. Probably something like https://github.com/guillemj/shadow/commit/b1476818de79ff74c8250a6fd04e65b2a52cedff

Reply:
Thanks!  Will try :)